docs/en/query_language/functions/type_conversion_functions.md

```diff
@@ -392,18 +392,77 @@ SELECT
 └───────────────────────────┴──────────────────────────────┘
 ```
 
-## parseDateTimeBestEffort {#type_conversion_functions-parsedatetimebesteffort}
+## parseDateTimeBestEffort {#parsedatetimebesteffort}
 
-Parse a number type argument to a Date or DateTime type.
-different from toDate and toDateTime, parseDateTimeBestEffort can progress more complex date format.
-For more information, see the link: [Complex Date Format](https://xkcd.com/1179/)
+Converts a string representation of a date and time value to its DateTime equivalent.
+Different from toDate and toDateTime, parseDateTimeBestEffort can progress more complex date format.
+
+**Syntax**
+```sql
+parseDateTimeBestEffort(s[, time_zone]);
+```
+
+**Parameters**
+
+- `s` — A string containing a date to convert. [String](../../data_types/string.md).  
+    The string to be parsed can take any of the following forms:
```

Comment:
I think we need a separate section for input formats description. Let's try to place it after Parameters and see how it looks.